pkgs/development/tools/git-ps-rs/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, libgpg-error
+, gpgme
+, dbus
+, openssl }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "git-ps-rs";
+  version = "6.5.0";
+
+  src = fetchFromGitHub {
+    owner = "uptech";
+    repo = "git-ps-rs";
+    rev = "${version}";
+    hash = "sha256-4wSm3H+98ZJZ+fZdLYshPKafRkPq98Pv3Lwh9o0be6U=";
+  };
+
+  cargoHash = "sha256-1p46xvo7abMPlVP8BeQ1j/8QQpK3kCgbTL3cdidfq04=";
+
+  nativeBuildInputs = [ pkg-config gpgme ];
+
+  buildInputs = [ openssl dbus libgpg-error ];
+
+  meta = with lib; {
+    description = "Tool for working with a stack of patches";
+    homepage = "https://git-ps.sh/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ alizter ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ alizter ];
    platforms = platforms.unix;
```

Reply:
Packages without platforms don't get built in the binary cache

Reply:
buildRustPackage already sets this